Components/Utils/ModelRenderer.cs

```diff
@@ -125,13 +126,36 @@ public HtmlString UnilingualPreFooter()
 
         public HtmlString PreFooter()
         {
+            Feedback feedback = new Feedback();
+            feedback.Enabled = _model.Settings.FeedbackLink.Show;
+
+            if (!string.IsNullOrWhiteSpace(_model.Settings.FeedbackLink.Text) || !string.IsNullOrWhiteSpace(_model.Settings.FeedbackLink.Section) || !string.IsNullOrWhiteSpace(_model.Settings.FeedbackLink.Theme))
+            {
+                feedback.Section = _model.Settings.FeedbackLink.Section;
+                feedback.Theme = _model.Settings.FeedbackLink.Theme;
+                if (Thread.CurrentThread.CurrentUICulture.TwoLetterISOLanguageName.StartsWith(Constants.FRENCH_ACCRONYM, StringComparison.OrdinalIgnoreCase))
+                {
+                    feedback.Text = _model.Settings.FeedbackLink.TextFr;
+                    feedback.Href = _model.Settings.FeedbackLink.UrlFr;
+                }
+                else
+                {
+                    feedback.Text = _model.Settings.FeedbackLink.Text;
+                    feedback.Href = _model.Settings.FeedbackLink.Url;
+                }
+            }
+            else if (!string.IsNullOrWhiteSpace(_model.Settings.FeedbackLink.Url))
```

Comment:
No problem with this necessarily, just a question: Is it ok to have Enabled = true but nothing else?

Reply:
Yup, if it is just Enabled = true, you just have a normal feedback tool without any contact stuff